src/reducers/filter.js

```diff
@@ -0,0 +1,8 @@
+const filterReducer = (state = '', action) => {
```

Comment:
- [ ] state should default to `all`

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/ItDefaultsToAll.png)